boards/px4/fmu-v5x/default.px4board

```diff
@@ -81,6 +81,7 @@ CONFIG_MODULES_NAVIGATOR=y
 CONFIG_MODE_NAVIGATOR_VTOL_TAKEOFF=y
 CONFIG_MODULES_PAYLOAD_DELIVERER=y
 CONFIG_MODULES_RC_UPDATE=y
+CONFIG_MODULES_ROVER_ACKERMANN=y
```

Comment:
I know that it was me proposing to add it to the v5x default - but I looks like it doesn't fit. I propose you keep it for now in the rover builds only, and we then in a follow up PR remove the legacy rover controller and then enable it on some default boards again if we can.